CruCentralCoast/Community/CommunityGroups/CommunityGroupDetailsVC.swift

```diff
@@ -32,6 +39,23 @@ class CommunityGroupDetailsVC: UIViewController {
         self.presentAlert(title: "Join Community Group", message: "Coming Soon...")
     }
     
+    @IBAction func didTapContactLeader() {
+        if (MFMessageComposeViewController.canSendText()) {
+            let controller = MFMessageComposeViewController()
+            controller.body = "Hey I'm interested in joining your community group!"
+            controller.recipients = leaderPhoneNumbers
+            controller.messageComposeDelegate = self
+            self.present(controller, animated: true, completion: nil)
+        }
+        else{print("error cant send text")}
+    }
+    
+    func messageComposeViewController(_ controller: MFMessageComposeViewController, didFinishWith result: MessageComposeResult) {
+        // Dismiss the message compose view controller.
```

Comment:
this comment pretty much says the same thing as the line of code below it, I'd just remove it